src/main/resources/GoogleApps/GoogleAppsConfigSheet.vm

```diff
@@ -13,82 +13,95 @@ $xwiki.ssx.use('GoogleApps.GoogleAppsConfigClass')
     $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}")##
 #end
 #macro(tp $n $p)##
-    $services.localization.render("GoogleApps.GoogleAppsConfigClass_${n}",$p)##
+    #set($prf="GoogleApps.GoogleAppsConfigClass_")
```

Comment:
Code style remarks, for readability (really helps review and development on other dev's code):
* for indentation, 2 spaces need to be used
* after `#set`, `#if`, `#foreach` we keep a blank space like in `#set (`
* `$prf` may stand for `prefix`, but why not using `$prefix` since it is so short?
* if there is no expression to evaluate, we use `'` for wrapping strings in velocity
These code style remarks are valid for the entire commit, I will not repeat them for each line.

Reply:
Done. Thanks.

Reply:
For prf, I renamed the two prefix variables.